internal/operators/lvm/config.go

```diff
@@ -7,13 +7,14 @@ import (
 const (
 	// LvmMinOpenshiftVersion is the minimum OCP version in which lvmo is supported
 	// Any changes here should be updated at line 16 too.
-	LvmMinOpenshiftVersion string = "4.12.0-0.0"
+	LvmMinOpenshiftVersion  string = "4.11.0"
```

Comment:
please change it to LvmoMinOpenshiftVersion

Reply:
done